ttyclock.c

```diff
@@ -567,6 +568,8 @@ main(int argc, char **argv)
      ttyclock.option.nsdelay = 0; /* -0FPS */
      ttyclock.option.blink = false;
 
+     ttyclock.rounding_permitted = false;
```

Comment:
i'm a bit confused as to why this is a setting that needs to be carried through everywhere.... it seems we only initialize it to false here, once, and then it remains true for the duration of the program, as soon as we have a "key event"... maybe some comments could clarify this as well?

Reply:
The reason is that initially, rounding is a bad thing because initially, the clock will not show the current time for the second half of every second with suppressing it. I agree that this implementation without at leat clarifying comments is sub par. Will improve and comment 🙂